src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/WithReferencesFindUsagesContext.cs

```diff
@@ -117,6 +117,21 @@ protected override Task OnReferenceFoundWorkerAsync(SourceReferenceItem referenc
                     addToEntriesWhenNotGroupingByDefinition: true);
             }
 
+            protected override Task OnExternalReferenceFoundWorkerAsync(ExternalReferenceItem reference)
+            {
+                // Normal references go into both sets of entries.
+                return OnEntryFoundAsync(
+                    reference.Definition,
+                    bucket => TryCreateExternalItemEntryAsync(reference),
+                    addToEntriesWhenGroupingByDefinition: true,
+                    addToEntriesWhenNotGroupingByDefinition: true);
+            }
+
+            private Task<Entry> TryCreateExternalItemEntryAsync(ExternalReferenceItem reference)
+            {
+                throw new NotImplementedException();
```

Comment:
note: will have to actually fill this out to properly create a window item populated from the info in this reference.